pkg/acsengine/defaults.go

```diff
@@ -624,10 +621,18 @@ func setMasterNetworkDefaults(a *api.Properties, isUpgrade bool) {
 	if a.MasterProfile == nil {
 		return
 	}
-
-	// Set default Distro to Ubuntu
-	if a.MasterProfile.Distro == "" {
-		a.MasterProfile.Distro = api.Ubuntu
+	o := a.OrchestratorProfile
+	// set default Distro based on orchestrator
+	switch o.OrchestratorType {
+	case api.OpenShift:
+		if a.MasterProfile.Distro == "" {
+			a.MasterProfile.Distro = api.OpenShiftLatestRHEL
```

Comment:
@jim-minter this is there latest default is being used. setting default if not specified. instead of copy pase we have "soft link" style pointer